README.md

```diff
@@ -1,22 +1,22 @@
-# Semantic Memory
+# Kernel Memory
 
-**Semantic Memory** (SM) is an open-source [service](dotnet/Service/README.md) and
+**Kernel Memory** (SM) is an open-source [service](dotnet/Service/README.md) and
```

Comment:
Do we still want to use "SM" as the abbreviation?

Reply:
KM, thanks!